core/constants.yaml

```diff
@@ -176,7 +176,7 @@ benchmark: &benchmark
   <<: *testnet
 
   genesisN: 50000
-  k: 6
+  k: 2
```

Comment:
No, it's not important.

The reason I made that I want to be more sensitive to bugs that happen at the transition from one epoch to the next.